cypress/integration/plugins/dashboards-assistant/conversation_history_spec.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { BASE_PATH } from '../../../utils/constants';
+import '@cypress/skip-test/support';
+
+if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
+  describe('Assistant conversation history spec', () => {
+    before(() => {
+      // Set welcome screen tracking to false
+      localStorage.setItem('home:welcome:show', 'false');
+      // Hide new theme modal
+      localStorage.setItem('home:newThemeModal:show', 'false');
+      // Visit OSD
+      cy.visit(`${BASE_PATH}/app/home`);
+      // Common text to wait for to confirm page loaded, give up to 60 seconds for initial load
+      cy.get(`input[placeholder="Ask question"]`, { timeout: 60000 }).should(
+        'be.length',
+        1
+      );
+
+      // Open chat flyout
+      cy.get('body')
```

Comment:
some of these look like we can add test subjects to the source code so that it is easier to find the values.

Reply:
This step is for check if flyout already opened before. Since we are visiting a new page. I think we can remove this check. The `flyout` must be closed for the initial status.